app/views/layouts/application.html.erb

```diff
@@ -90,24 +90,24 @@
   <div class="container">
     <hr>
 
-    <div class="row">
+    <div id="footer-authoring" class="row">
```

Comment:
That means, you are free to do whatever you want with your customizations, and there is nothing wrong with adding the ID. But this is a just a friendly warning about not doing bad things :stuck_out_tongue: